controllers/category.js

```diff
@@ -38,10 +39,101 @@ export default class CategoryController {
           errors: [error.errors[0].message]
         });
       }
-      return res.status(500).json({
-        success: false,
-        errors: ['Internal server error']
+      return res.status(500).json({ success: false, error: [error.message] });
+    }
+  }
+
+  /**
+    * @description - Get all category
+    * @static
+    * @param {object} req - HTTP Request
+    * @param {object} res - HTTP Response
+    * @memberof CategoryController
+    * @returns {object} response object
+    */
+  static async getAllCategory(req, res) {
```

Comment:
`getAllCategories`

Reply:
In our last demo, the PO wanted to see evidence that a deleted category no longer exists in the system. So I implemented the `getAllCategories` endpoint to allow for that instance.